recipes/dlo-hic-tools/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+    name: dlo_hic
+    version: "0.0.0"
+
+source:
+    url: https://github.com/Nanguage/DLO-HiC-Tools/archive/v0.0.0.tar.gz # [unix and py3k]
+    sha256: 5caddccd25fb4f86689dc8b6e85f4eb4a3de990766ddb47670097671cb403b71 # [unix and py3k]
+
+build:
+    number: 1
+    skip: True # [py27]
+
+requirements:
+    host:
+        - python
+        - cython
+        - setuptools
+    run:
+        - bwa
+        - samtools
+        - pairix
+        - tabix
+        - python
+        - numpy
+        - scipy
+        - matplotlib
+        - biopython
+        - click
+        - seaborn
+        - pyfaidx
+
+test:
+    commands:
+        dlohic --help
+    imports:
+        - dlo_hic
+        - dlo_hic.api
+        - dlo_hic.tools.main_process
+        - dlo_hic.tools.helper
+        - dlo_hic.tools.quality_control
+        - dlo_hic.utils.align
+
+about:
+    home: https://github.com/Nanguage/DLO-HiC-Tools
+    license: GPL3
```

Comment:
You could make this
```yaml
  license: GPL-3.0-only
```
to get the less ambiguous identifier from https://github.com/spdx/license-list-data/blame/v3.2/licenses.md#L165.

BTW: You should fix the license classifier upstream at  https://github.com/Nanguage/DLO-HiC-Tools/blob/v0.0.0/setup.py#L51.